jsonrpc/filter_manager.go

```diff
@@ -341,8 +366,17 @@ func (f *FilterManager) GetFilterChanges(id string) (string, error) {
 }
 
 func (f *FilterManager) Uninstall(id string) bool {
```

Comment:
Why do we need the feedback if it was uninstalled?

Reply:
https://github.com/0xPolygon/polygon-edge/blob/876489ae0cef41dab883463cffe74ebb6e658a34/jsonrpc/dispatcher.go#L195

https://github.com/0xPolygon/polygon-edge/blob/876489ae0cef41dab883463cffe74ebb6e658a34/jsonrpc/dispatcher.go#L167

Reply:
Jesus Christ 🤦 

Resolving because we will clean this trash up later...